docs/layers/geojson-layer.md

```diff
@@ -157,13 +157,13 @@ deck.gl.
 - Default: `f => f.properties.lineColor || [0, 0, 0, 255]`
 
 Called to retrieve the color of line and/or the outline of polygon color for a GeoJson feature, depending
-on its type. Color supports `[R, G, B, A]` and `[R, G, B, A]`, where `A` is scaled to a percentage between `0` and `255`.
+on its type. Color supports `[R, G, B]` and `[R, G, B, A]`, where `A` is scaled to a percentage between `0` and `255`.
```

Comment:
While we are at it, we should probably drop the word `percentage` as it has strong associations with the range 0-100. Replace `scaled to a percentage` with `has a value between`?

Reply:
Sure thing